pkg/project/admission/nodeenv/admission.go

```diff
@@ -26,8 +26,8 @@ type podNodeEnvironment struct {
 
 // Admit enforces that pod and its project node label selectors matches at least a node in the cluster.
 func (p *podNodeEnvironment) Admit(a admission.Attributes) (err error) {
-	// ignore anything except create or update of pods
-	if !(a.GetOperation() == admission.Create || a.GetOperation() == admission.Update) {
+	// ignore anything except pod creation
+	if a.GetOperation() != admission.Create {
 		return nil
 	}
```

Comment:
need to only run this check if `a.GetSubresource() == ""`, otherwise this could block subresources that support create (`pods/binding` comes to mind, not sure what else)